src/Skills.jsx

```diff
@@ -188,36 +189,64 @@ class Skills extends Component {
     return null;
   }
 
+  loadReviews() {
+    const currentId = this.state.currentSkill.skill_id;
+    return axios.get(`${api}/skillreviews?skill_id=${currentId}`)
+      .then(res => {
+        const results = res.data;
+        this.setState({
+          reviews: results
+        })
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  }
+
   render() {
     const onSkillChange = ev => this.onChange("skill_id", ev.id);
     const currentSkillID = this.state.currentSkill.skill_id;
     const isSkillSelected = currentSkillID !== "";
+    let reviews = null;
+    if (this.state.reviews){
+      reviews = this.state.reviews.map(review => {
+        let header = review.team_member_name + " on " + new Date(review.timestamp).toDateString();
+        return (<Panel header={header} key={review.id}>{review.body}</Panel>);
+      });
+    }
+    const reviewList = (
+      <div>{reviews}</div>
+    );
     let display = null;
     if (isSkillSelected) {
       display = (
-        <SelectedItem
-          typeName="Skill"
-          deleteCallback={this.openDeleteModal}
-        >
-          <h1>{this.state.currentSkill.name}</h1>
-          <h4>{this.state.currentSkill.skill_type}</h4>
-          {this.makeLinks()}
-          <Button
-            name="AddLink"
-            bsStyle="primary"
-            onClick={this.openLinkModal}
+        <div>
+          <SelectedItem
+            typeName="Skill"
+            deleteCallback={this.openDeleteModal}
           >
-            Add Link
-          </Button>
-          <Button
-            name="AddReview"
-            bsStyle="primary"
-            onClick={this.openReviewModal}>
-            Add Review
-          </Button>
-        </SelectedItem>
+            <h1>{this.state.currentSkill.name}</h1>
+            <h4>{this.state.currentSkill.skill_type}</h4>
+            {this.makeLinks()}
+            <Button
+              name="AddLink"
+              bsStyle="primary"
+              onClick={this.openLinkModal}
+            >
+              Add Link
+            </Button>
+            <Button
+              name="AddReview"
+              bsStyle="primary"
+              onClick={this.openReviewModal}>
+              Add Review
+            </Button>
+          </SelectedItem>
+          {reviewList}
+        </div>
```

Comment:
You can put the `reviewList` into `<SelectedItem />`. It would save having to have a `<div>` to encapsulate the two, and the purpose of `<SelectedItem />` is to contain things that should only appear if there is a selected item